.circleci/template.sh

```diff
@@ -1,2 +1,3 @@
 #!/usr/bin/env bash
 export CIRCLECI_TEMPLATE=go-distribution-oss
+export PUBLISH_WITH_GRADLE=true
```

Comment:
See the internal https://github.p.b/excavator-checks/circle-templates/pull/900/ for where this is added.